lib/FusionInventory/Agent/Task/Inventory/Virtualization.pm

```diff
@@ -9,6 +9,15 @@ sub isEnabled {
     return 1;
 }
 
+our $STATUS_RUNNING = 'running';
+our $STATUS_BLOCKED = 'blocked';
+our $STATUS_IDLE = 'idle';
+our $STATUS_PAUSED = 'paused';
+our $STATUS_SHUTDOWN = 'shutdown';
+our $STATUS_CRASHED = 'crashed';
+our $STATUS_DYING = 'dying';
+our $STATUS_OFF = 'off';
+
```

Comment:
Personally, I would prefer a hash here in place of x scalar or even some "use constant" declaration & export of them. You should also check if this could be re-used in other sub-classes.